### PR TITLE
[FASTLY] Update fastly spec to 0.30.0

### PR DESCRIPTION
**Pull request by kailan**
0.30.0 adds a `fastly compute pack` command for packaging compiled binaries into Fastly-ready packages.

### CODE REVIEW

**Comment by kailan** [comment]
I think your linting rules have changed, hence the massive diff on the js file.

File: dev/fastly.ts [hunk 1]
```diff
@@ -610,6 +610,23 @@ export const completionSpec: Fig.Spec = {
             },
           ],
         },
+        {
+          name: "pack",
+          description:
+            "Package a pre-compiled Wasm binary for a Fastly Compute@Edge service",
+          subcommands: [],
```
Comment (QuiiBz) [inline dev/fastly.ts:617]: You can safely remove `subcommands` since it's empty.

**Review by QuiiBz — CHANGES_REQUESTED** [review]

File: dev/fastly.ts (hunk 1 above)
Comment (cstrnt) [reply dev/fastly.ts:617]: #205 will fix this :)

**Review by cstrnt — APPROVED** [review]
Apart from Tom's comment this looks good to me :)

**Comment by kailan** [comment]
Cool, I'll wait on #205 and then rebase this PR on the new changes.

**Comment by mattschrage** [comment]
@kailan just merged #205 - you're good to go!

File: dev/fastly.ts [hunk 2]
```diff
@@ -590,6 +590,23 @@ export const completionSpec: Fig.Spec = {
             },
           ],
         },
+        {
+          name: "pack",
+          description:
+            "Package a pre-compiled Wasm binary for a Fastly Compute@Edge service",
+
+          options: [
```
Comment (QuiiBz) [inline dev/fastly.ts:598]: Looks like an empty line as popped here, could you remove it?

Comment (kailan) [reply dev/fastly.ts:598]: Every other entry in the file is like this – this must have been changed in the recent linting update.

I didn't add the line break myself, it must have been done by the linting tool when I ran `--fix`.

Comment (QuiiBz) [reply dev/fastly.ts:598]: Strange, we will look into this. Let's merge this!

**Review by QuiiBz — APPROVED** [review]

**Pull request merged**